packages/core/src/browser/keybinding.spec.ts

```diff
@@ -456,31 +456,12 @@ const TEST_COMMAND_SHADOW: Command = {
 @injectable()
 export class TestContribution implements CommandContribution, KeybindingContribution {
 
-    constructor(@inject(KeybindingContextRegistry) protected readonly contextRegistry: KeybindingContextRegistry) {
-    }
-
     registerCommands(commands: CommandRegistry): void {
         commands.registerCommand(TEST_COMMAND);
         commands.registerCommand(TEST_COMMAND2);
         commands.registerCommand(TEST_COMMAND_SHADOW);
     }
 
-    registerContexts() {
-        this.contextRegistry.registerContext(
-            {
-                id: 'testContext',
-                isEnabled(arg?: Keybinding): boolean {
-                    return true;
-                }
-            },
-            {
-                id: 'testContext',
-                isEnabled(arg?: Keybinding): boolean {
-                    return true;
-                }
-            },
```

Comment:
Presumably, this duplicate context registration (with the same IDs) was to check whether it throws an error or not, right? Since you got rid of the error throwing part (and you just log it, which is more graceful) you have removed this too, correct?

Reply:
I've tried, tests were green regardless of it.

Reply:
Me too, I was just asking whether you know exactly why we had that or not. I am OK with the removal.